### DISCUSSION TITLE
Peptidomics

### DISCUSSION THREAD

**Discussion by SauerTho** [vdemichev/DiaNN#245] [Q&A]
Hey,
I was wondering if there is a possibility to do peptidomics analysis without protease digest with DIA-NN? We have some sample where peptides are isolated via specific columns based on their MHC binding capacity. However, they are digested via Trypsin/P.
I understand that some of the major advantages of DIA-NN might get damped.

Cheers
Thorben

**Comment by vdemichev** [comment]
Hi Thorben,

Yes, just format the list of peptides as a FASTA database:
>pep1
PEPTIDE
>pep2
PEPTIDEK
>pep3
PEPTIDEKR
...

And write "-- cut " (without quotes) in the 'Additional options' textbox.

Best,
Vadim
